stan/math/opencl/sub_block.hpp

```diff
@@ -59,7 +59,40 @@ inline void matrix_cl<T, enable_if_arithmetic<T>>::sub_block(
   } else {
     opencl_kernels::sub_block(cl::NDRange(nrows, ncols), A, *this, A_i, A_j,
                               this_i, this_j, nrows, ncols, A.rows(), A.cols(),
-                              this->rows(), this->cols(), triangular_view);
+                              this->rows(), this->cols(), A.triangular_view());
+  }
+  // calculation of extreme sub- and super- diagonal written
+  int diag_in_copy = A_i - A_j;
+  int copy_low
+      = containsNonzeroPart(A.triangular_view_, TriangularViewCL::Lower)
```

Comment:
I would prefer underscores over CamelCase, or does CamelCase have precedence somewhere else in the stan library for this sort of function